src/sodaspark/scan.py

```diff
@@ -296,4 +303,136 @@ def execute(
     df.createOrReplaceTempView(scan_yml.table_name)
     scan = create_scan(scan_yml, soda_server_client=soda_server_client)
     scan.execute()
+    if as_frame:
```

Comment:
Could you rewrite this to have one `return`? That is a style choice I prefer

``` python
    if as_frames:
        result = convert_scan_result_to_spark_data_frames(scan.scan_result)
    else:
         result = scan.scan_result
    return result
```

Reply:
`return` is rewritten.